app/views/admin/shared/_admin_nav.html.erb

```diff
@@ -5,7 +5,9 @@
     <%= link_to "Update Account", edit_user_path(current_user), class: "nav-item nav-link" %>
     <%= link_to "My Stores", user_stores_path, class: "nav-item nav-link" %>
     <%= link_to "My Items", admin_items_path, class: "nav-item nav-link" %>
-    <%= link_to "Manage Employees", admin_employees_path, class: "nav-item nav-link" %>
+    <% if current_user.store_admin? %>
```

Comment:
awesome. I really like how this continues to differentiate based on the admin role!